test/plugins/auth.test.js

```diff
@@ -485,14 +486,34 @@ describe('auth plugin test', () => {
             })
         );
 
-        it('fails to issue a jwt given an invalid application auth token', () =>
+        it('returns user signed token given an API access token', () =>
             server.inject({
+                url: `/auth/token?access_key=${apiKey}`
```

Comment:
> Is there a possibility that the temporaryAccessKey and apiKey could collide?

Yes. Once we fully support the app tokens, we should switch away from the `temporaryAccessKey`.